.pre-commit-config.yaml

```diff
@@ -735,22 +692,6 @@ repos:
 
 
   # ----- Cloud CI Requirements ------------------------------------------------------------------------------------->
-      - id: pip-tools-compile
```

Comment:
The current cloud tests are running on python 3.6
https://jenkins.saltproject.io/job/pr-centos-7-x86_64-py3-cloud-pytest/job/master/4029/consoleFull